ablySpec/RealtimeClientChannel.swift

```diff
@@ -552,6 +552,52 @@ class RealtimeClientChannel: QuickSpec {
 
                 }
 
+                // RTL6i
+                context("expect either") {
+
+                    pending("an array of Message objects") {
+                        let options = AblyTests.commonAppSetup()
+                        options.autoConnect = false
+                        let client = ARTRealtime(options: options)
+                        client.setTransportClass(TestProxyTransport.self)
+                        client.connect()
+                        defer { client.close() }
+                        let channel = client.channels.get("test")
+
+                        let messages = [ARTMessage(data: ["value":1], name: nil), ARTMessage(data: ["value":2], name: nil)]
+                        var result: [ARTMessage]?
+
+                        channel.subscribe { message in
+                            result = message.data as? [ARTMessage]
```

Comment:
I think I've asked this before but don't think I got an answer, and unfortunately I just don't understand this. How can `expect(result).toEventually(equal(messages))` when result is being assigned with a `=` operator here i.e. it can only ever have one value?